.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
For me it is unclear what is actually being used for this github workflow, I see no "run" statements, but maybe I am missing something...

also we have no branch named main